spec/support/crypt.rb

```diff
@@ -52,6 +52,13 @@ module Crypt
 
     # Example value to encrypt
     let(:ssn) { '123-456-7890' }
+
+    let(:key_vault_collection) do
+      authorized_client.with(
+        database: key_vault_db,
+        write_concern: { w: :majority }
+      )[key_vault_coll]
+    end
```

Comment:
From the [prose tests](https://github.com/mongodb/specifications/tree/master/source/client-side-encryption/tests): Perform all applicable operations on key vault collections (e.g. inserting an example data key, or running a find command) with readConcern/writeConcern "majority".